src/components/elements/Map-mapbox/components/OverviewMapArea.tsx

```diff
@@ -25,12 +27,27 @@ const OverviewMapArea = ({ entityModel, type }: EntityAreaProps) => {
   const mapFunctions = useMap();
   const [checkedValues, setCheckedValues] = useState<string[]>([]);
   const [sortOrder, setSortOrder] = useState<string>("created_at");
+  const { isMonitoring } = useMonitoringPartner();
+  const { data: entityData, refetch } = useGetV2TypeEntity({
+    queryParams: {
+      uuid: entityModel?.uuid,
+      type: type,
+      status: checkedValues.join(","),
+      [`sort[${sortOrder}]`]: sortOrder === "created_at" ? "desc" : "asc"
+    }
+  });
   const setResultValues = (result: any) => {
-    if (result.polygonsData) {
+    if (result?.polygonsData) {
       setPolygonsData(result.polygonsData);
       setEntityBbox(result.bbox as BBox);
+      if (result.polygonsData?.length === 0) {
+        callCountryBBox();
+      }
     }
   };
+  const recallEntityData = () => {
+    refetch();
+  };
```

Comment:
🎨 Seems like `refetch` could just be passed down to the child components below instead of creating a function that does nothing but call it.

Reply:
done